compiled-lang/es.json

```diff
@@ -0,0 +1,308 @@
+            {
+              "type": 0,
+              "value": "Eliminar"
+            }
+          ]
+        }
+      },
+      "pluralType": "cardinal",
+      "type": 6,
+      "value": "isDeleting"
+    }
+  ],
+  "pickers.button.save": [
+    {
+      "offset": 0,
+      "options": {
+        "one": {
+          "value": [
+            {
+              "type": 0,
+              "value": "Cargando..."
+            }
+          ]
+        },
+        "other": {
+          "value": [
+            {
+              "type": 0,
+              "value": "Guardar"
+            }
+          ]
+        }
+      },
+      "pluralType": "cardinal",
+      "type": 6,
+      "value": "isLoading"
+    }
+  ],
+  "pickers.create.button.text": [
+    {
+      "type": 0,
+      "value": "Agregar Nuevo Recolector"
+    }
+  ],
+  "pickers.create.form.address.label": [
+    {
+      "type": 0,
+      "value": "Dirección"
+    }
+  ],
+  "pickers.create.form.blood_type.label": [
+    {
+      "type": 0,
+      "value": "Tipo de Sangre"
+    }
+  ],
+  "pickers.create.form.emergency_contact_name.label": [
+    {
+      "type": 0,
+      "value": "Nombre de Contacto de Emergencia"
+    }
+  ],
+  "pickers.create.form.emergency_contact_number.label": [
+    {
+      "type": 0,
+      "value": "Número de Contacto de Emergencia"
+    }
+  ],
+  "pickers.create.form.identification_number.label": [
+    {
+      "type": 0,
+      "value": "Número de Identificación"
+    }
+  ],
+  "pickers.create.form.name.label": [
+    {
+      "type": 0,
+      "value": "Nombre"
+    }
+  ],
+  "pickers.create.form.phone_number.label": [
+    {
+      "type": 0,
+      "value": "Número de Teléfono"
+    }
+  ],
+  "pickers.create.form.relation_to_picker.label": [
+    {
+      "type": 0,
+      "value": "Relación con el recolector"
+    }
+  ],
+  "pickers.create.picker.response.error": [
+    {
+      "type": 0,
+      "value": "¡Ups! No se pudo guardar el recolector."
+    }
+  ],
+  "pickers.create.picker.response.success": [
+    {
+      "type": 0,
+      "value": "El recolector se creó correctamente"
+    }
+  ],
+  "pickers.delete.label": [
+    {
+      "type": 0,
+      "value": "Eliminar datos del recolector"
+    }
+  ],
+  "pickers.delete.picker.response.error": [
+    {
+      "type": 0,
+      "value": "¡Ups! No se pudo eliminar el recolector."
+    }
+  ],
+  "pickers.delete.picker.response.success": [
+    {
+      "type": 0,
+      "value": "El recolector se eliminó correctamente."
+    }
+  ],
+  "pickers.detail.address.label": [
+    {
+      "type": 0,
+      "value": "Dirección"
+    }
+  ],
+  "pickers.detail.blood_type.label": [
+    {
+      "type": 0,
+      "value": "Tipo de Sangre"
+    }
+  ],
+  "pickers.detail.button.view_harvest_log": [
+    {
+      "type": 0,
+      "value": "Ver Registro de Temporada"
+    }
+  ],
+  "pickers.detail.emergency_contact_person.label": [
+    {
+      "type": 0,
+      "value": "Persona de Contacto de Emergencia"
```

Comment:
Maybe "Nombre Contacto de Emergencia" sounds better, or just "Contacto de Emergencia"

Reply:
We will get to polish the translation before the beta release